plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewNativeAppEngineStandardProjectTest.java

```diff
@@ -75,6 +76,7 @@ private void createAndCheck(String projectName, String location, String packageN
     assertTrue(project.exists());
 
     IFacetedProject facetedProject = new FacetedProjectHelper().getFacetedProject(project);
+    assertNotNull("Native App Engine projects are faceted", facetedProject);
```

Comment:
"Native App Engine projects should be faceted" may seem less confusing when later one sees this error message.